package.json

```diff
@@ -63,6 +63,7 @@
     "type-check": "tsc -p . && tsc -p lighthouse-viewer/",
     "i18n:checks": "./lighthouse-core/scripts/i18n/assert-strings-collected.sh",
     "i18n:collect-strings": "node lighthouse-core/scripts/i18n/collect-strings.js",
+    "update:lantern": "node lighthouse-core/scripts/lantern/update-master-lantern-values.js",
```

Comment:
it doesn't really matter, but `update:lantern-master` or something since we aren't allowed comments in `package.json`?

Reply:
sure, done